### PR TITLE
migration: Remove broken hearing (FPLA-2525)

### PR DESCRIPTION
**Pull request by tomaszpowroznik**
https://tools.hmcts.net/jira/browse/FPLA-2525
https://github.com/hmcts/fpl-ccd-data-migration-tool/pull/37/files

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java [hunk 1]
```diff
@@ -41,62 +38,48 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
...
-            if (hearings.size() < 5) {
-                throw new IllegalArgumentException(
-                    format("Case has %s hearing(s), expected at least %s", hearings.size(), 5));
+            if (hearings.isEmpty()) {
+                throw new IllegalArgumentException("No hearings in a case");
             }
 
-            Element<HearingBooking> hearing1 = hearings.get(3);
-            Element<HearingBooking> hearing2 = hearings.get(4);
+            Element<HearingBooking> hearingToBeRemoved = hearings.get(0);
```
Comment (joez-f) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java:62]: are we sure it is first hearing in the list? I am guessing yes

**Review by joez-f — APPROVED** [review]
looks good

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java [hunk 2]
```diff
@@ -71,115 +61,126 @@
...
-        Element<HearingBooking> hearing1 = buildHearing(2020, NOVEMBER, 19, CASE_MANAGEMENT);
-        Element<HearingBooking> hearing2 = buildHearing(2021, FEBRUARY, 5, FURTHER_CASE_MANAGEMENT);
-        Element<HearingBooking> hearing3 = buildHearing(2021, MARCH, 4, ISSUE_RESOLUTION);
-        Element<HearingBooking> hearing4 = buildHearing(2021, FEBRUARY, 5, FURTHER_CASE_MANAGEMENT);
-        Element<HearingBooking> hearing5 = buildHearing(2021, MARCH, 4, ISSUE_RESOLUTION);
+    class Fpla2525 {
+        String familyManCaseNumber = "SN20C50028";
+        String migrationId = "FPLA-2525";
+
+        Element<HearingBooking> hearing1 = element(HearingBooking.builder()
```
Comment (lewisbirks) [inline service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java:68]: There are unused buildHearing methods

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java (hunk 1 above)
Comment (tomaszpowroznik) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java:62]: yes, this is the only one.

**Pull request merged**